### PR TITLE
removed link underlining

### PR DESCRIPTION
**Pull request by alagrin92**
Test of update to files

### CODE REVIEW

**Comment by marlonkeating** [comment]
Looks reasonable at first glance, but I'm still seeing link underlining here:
https://democracy-lab-staging-pr-25.herokuapp.com/index/?section=FindProjects

File: common/components/componentsBySection/FindProjects/ProjectCard.jsx [hunk 1]
```diff
@@ -21,7 +25,7 @@ class ProjectCard extends React.PureComponent<Props> {
 
   render(): React$Node {
     return (
-      <a
+      <a style={styles}
```
Comment (marlonkeating) [inline common/components/componentsBySection/FindProjects/ProjectCard.jsx]: Is there a need to inject this style into the markup?

File: package.json [hunk 2]
```diff
@@ -23,15 +21,17 @@
     "react": "^16.0.0",
     "react-bootstrap": "^0.31.5",
     "react-dom": "^16.0.0",
-    "react-select": "^1.2.0",
+    "react-select": "^1.2.1",
```
[the remark this answers is not in the document]
Comment (alagrin92) [reply package.json]: this one fixed the errors I was getting upon trying to run the build

File: common/components/componentsBySection/FindProjects/ProjectCard.jsx (hunk 1 above)
Comment (alagrin92) [reply common/components/componentsBySection/FindProjects/ProjectCard.jsx]: This type of fix was the only one that actually was making sure all links at the same time were not underlining, unless there is a better method. Tried all I knew!

Comment (marlonkeating) [reply common/components/componentsBySection/FindProjects/ProjectCard.jsx]: Alright.  In that case can we remove the redundant lines in the .css?

Comment (alagrin92) [reply common/components/componentsBySection/FindProjects/ProjectCard.jsx]: Will do!

**Pull request merged**